HacktoberfestProject.Web/Views/Shared/_Layout.cshtml

```diff
@@ -42,6 +42,8 @@
 				<p class="text-base text-gray-700">&copy; 2020 - PR Tracker for .NET</p>
 		</footer>
 
+		<script src="~/lib/jquery/dist/jquery.min.js"></script>
+		<script src="~/lib/typeahead.js/typeahead.bundle.min.js"></script>
```

Comment:
Do we want to load this on every view, or just the ones we need it on? It could go in the Scripts section of a view.

Reply:
I would be happy to move this on a follow-up if you would like @Layla-P ?